### PR TITLE
Banner to map page

### PR DESCRIPTION
**Pull request by IlyasAli1**
https://trello.com/c/Zj3Rex2D/70-add-banner-image-to-map-page

Also removed margin from the top of the Map component so that there isn't so much space above the "Map of Sightings" header.

### CODE REVIEW

**Comment by IlyasAli1** [comment]
Screenshot (24)

File: WhaleSpotting/react-app/src/styles/Map.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import "./Constants.scss";
 
 .map-component {
-    margin: $headerHeight auto $footerHeight auto;
+    margin: 0 auto $footerHeight auto;
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/Map.scss]: I don't think I agree with doing this, I think we should extract the margin stuff to the parent div surrounding the banner, as the banner is now too high up

Comment (oskarwilliams) [reply WhaleSpotting/react-app/src/styles/Map.scss]:
```suggestion
    margin: 0 auto;
```

File: WhaleSpotting/react-app/src/styles/Map.scss [hunk 2]
```diff
@@ -1,7 +1,11 @@
 @import "./Constants.scss";
 
+.map-component-container {
+    margin-top: $headerHeight;
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/src/styles/Map.scss:4]:
```suggestion
    margin-top: $headerHeight;
    padding-bottom: $footerHeight;
```

Comment (oskarwilliams) [reply WhaleSpotting/react-app/src/styles/Map.scss:4]: Can you add this mobile styling as well

**Review by oskarwilliams — CHANGES_REQUESTED** [review]

File: WhaleSpotting/react-app/src/styles/Map.scss (hunk 1 above)
Comment (IlyasAli1) [reply WhaleSpotting/react-app/src/styles/Map.scss]: Done

File: WhaleSpotting/react-app/src/styles/Map.scss (hunk 2 above)
Comment (IlyasAli1) [reply WhaleSpotting/react-app/src/styles/Map.scss:4]: Done

**Review by oskarwilliams — APPROVED** [review]

**Pull request merged**